releasenotes/notes/add-logging-171aceb37c21a3d5.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Added logging utility functions accessible through `qiskit_nature.settings.logging`.
```

Comment:
That's not how it currently appears in the documentation. There you expose `QiskitNatureLogging` directly on the root so we should unify this.
I'm fine with either exposing this as part of `settings` or via it's own singleton instance (e.g. `qiskit_nature.logging`).

Reply:
I understood the issue as being exposed through `qiskit_nature.logging`. The reason for exposing `QiskitNatureLogging` in the root was because I could not find a way to show the sphinx documentation for this calss otherwise. Suggestions are welcome :-)

Reply:
Okay I see :thinking: I also think `qiskit_nature.logging` makes more sense but the reno says otherwise..

Reply:
I changed to qiskit_nature.logging